source/reactjs/README.md

```diff
@@ -1,10 +1,29 @@
+# MSApprovals
+MSApprovals is an important service and platform that supports 16000 users in approving ~750K approvals each month with the total amount ranging anywhere from ~3B$ to ~5B$ and this is across ~21 different Line of Business applications.
```

Comment:
Let's use MS Assent and also swap this content for something which we already have in the other readme file.

Reply:
I've updated the readme accordingly

Reply:
Microsoft Assent (a.k.a Approvals) as a platform provides the "one stop shop" solution for approvers via a model that brings together disparate different approval requests in a consistent and ultra-modern model. Approvals delivers a unified approvals experience for any approval on multiple form factors - Website, Outlook Actionable email, Teams. It consolidates approvals across organization's line of business applications, building on modern technology and powered by Microsoft Azure. It serves as a showcase for solving modern IT scenarios using the latest technologies.